tests/test_operation.py

```diff
@@ -418,6 +418,19 @@ class DummyOp(oo.Operation):
         with self.assertRaisesRegex(ValueError, "Must specify the wires"):
             DummyOp(0.54, 0, do_queue=False)
 
+    def test_observable_return_type_none(self):
+        """Test exception raised if no return type is specified"""
```

Comment:
Does not appear to test for an exception, but rather that the returned value of the dummy observable is `None`